packages/edit-site/src/components/layout/router.js

```diff
@@ -94,21 +96,34 @@ export default function useLayoutAreas() {
 						content={ <DataViewsSidebarContent /> }
 					/>
 				),
-				content: <PostList postType={ postType } />,
-				preview: ! showQuickEdit &&
-					( isListLayout || hasEditCanvasMode ) && <Editor />,
+				content:
+					showQuickEdit && isListLayout && !! postId ? (
+						<PostEdit
+							postType={ postType }
+							postId={ postId }
+							onBack={ () => {
+								history.push( {
+									postId,
```

Comment:
Same comment about location params and set quickEdit to `undefined`.

Reply:
Why set "quickEdit" to undefined?

Reply:
Don't remember what I tested before (something like quick edit and then change view..), but now I retested and didn't notice something weird.